nginx_stage/lib/nginx_stage/session_finder.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+module NginxStage
+  module SessionFinder
+    def session_count(user)
+      `ps  -o cmd -u #{user}`.split("\n").select do |command|
+        command.match?(/Passenger \w+App:/)
```

Comment:
Might be good to leave comment about what this could match, as I presume you want to match like `Passenger RubyApp:` or `Passenger NodeApp:`.  Would be interesting to see if this match works for non-Ruby and non-NodeJS apps, such as Python apps maybe.